README.md

```diff
@@ -746,6 +746,74 @@ or Windows:
 ./gradlew.bat build
 ```
 
+# Testing
+
+To run the tests, you need the following tools installed:
+   * Unity (with iOS and Android modules installed)
+
+You can run the tests by running the following from your shell
+(Linux / OSX):
+
+```
+./gradlew test
+```
+
+or Windows:
+
+```
+./gradlew.bat test
+```
+
+The following properties can be set to narrow down the tests to run or change
+the test run behavior.
+
+* `INTERACTIVE_MODE_TESTS_ENABLED` - Default to `1`. Set to `1` to enable
+  interactive mode tests, which requires GPU on the machine. Otherwise, only run
+  tests in the batch mode.
+* `INCLUDE_TEST_TYPES` - Default to empty string, which means to include every
+  type of the test. To narrow down the types of test to run, set this properties
+  with a list of case-insensitive type strings separated by comma. For instance,
+  `-PINCLUDE_TEST_TYPES="Python,NUnit"` means to include only Python tests and
+  NUnit tests.
+  See `TestTypeEnum` in `build.gradle` for availble options.
```

Comment:
available* (this happens a couple times in this)

Reply:
Good catch. Thank you.